src/ggcomm.c

```diff
@@ -1344,6 +1346,26 @@ concatenate_results (RESULT_NODE *r,
 
     rc [totlines * totlinelength] = 0;
 
+    /*  Trim whitespace from all lines  */
+    txt = rc;
+    while (txt)
+      {
+         line_end = strchr(txt, '\n');
+         if (!line_end)
+           break;
+         if (*(line_end - 1) == '\r')
```

Comment:
@evoskuil there's a ```file_crlf``` global variable but it doesn't seem to work, at least in this context.
This solution seems to do the trick (with the change at line 1366 as well).
What do you think?